src/stactools/palsar/constants.py

```diff
@@ -1,20 +1,62 @@
 from pystac import Link, Provider
+from pystac import ProviderRole as PR
+# from pystac.extensions.sar import
+from pystac.extensions.eo import Band
 
 ALOS_PALSAR_PLATFORM = "alos-2/alos"
 ALOS_PALSAR_INSTRUMENTS = ["PALSAR-2", "PALSAR"]
 ALOS_PALSAR_GSD = 25  # meters
 ALOS_PALSAR_EPSG = 4326
 ALOS_PALSAR_PROVIDERS = [
     Provider("Japan Aerospace Exploration Agency",
-             roles=["producer", "processor", "licensor"],
+             roles=[PR.PRODUCER, PR.PROCESSOR, PR.LICENSOR],
              url="https://www.eorc.jaxa.jp/ALOS/en/dataset/fnf_e.htm"),
+    Provider("Microsoft Planetary Computer", roles=[PR.HOST])
 ]
 ALOS_PALSAR_LINKS = [
     Link(
         "handbook",
-        "https://www.eorc.jaxa.jp/ALOS/en/dataset/pdf/DatasetDescription_PALSAR2_Mosaic_FNF_revK.pdf",
+        "https://www.eorc.jaxa.jp/ALOS/en/dataset/pdf/DatasetDescription\
+            _PALSAR2_Mosaic_FNF_revK.pdf",
         "application/pdf",
-        "Global 25 m Resolution PALSAR-2/PALSAR Mosaic and Forest/Non-Forest Map (FNF) Dataset Description",
+        "Global 25 m Resolution PALSAR-2/PALSAR Mosaic and Forest/Non-Forest Map (FNF) \
+            Dataset Description",
         extra_fields={"description": "Also includes data usage information"})
 ]
-ALOS_PALSAR_BANDS = {}
+ALOS_PALSAR_BANDS = {
+    1:
+    Band.create(
+        common_name="HH",
```

Comment:
I trust there are no pystac "builtins" for these?

Reply:
I think the Band class is the builtin. Note that Bands are part of the stac eo extension, not the core. Might end up dropping later anyways. Also addressed in another branch.